src/OrbitGl/include/OrbitGl/TrackManager.h

```diff
@@ -115,8 +116,8 @@ class TrackManager {
 
   std::vector<std::shared_ptr<Track>> all_tracks_;
   absl::flat_hash_map<uint32_t, std::shared_ptr<ThreadTrack>> thread_tracks_;
-  std::map<std::string, std::shared_ptr<AsyncTrack>> async_tracks_;
-  std::map<std::string, std::shared_ptr<VariableTrack>> variable_tracks_;
+  std::map<std::string, std::shared_ptr<AsyncTrack>, std::less<>> async_tracks_;
+  std::map<std::string, std::shared_ptr<VariableTrack>, std::less<>> variable_tracks_;
```

Comment:
This change is needed for transparent comparison (for `tracks_.find()` to accept a `std::string_view` even though the key type is `std::string`).